servicepulse/intro-archived-messages.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Archived Message Management in ServicePulse
+summary: Describes how ServicePulse monitors archived failed messages, and allows un-archiving archived failed messages
+tags:
+- ServicePulse
+reviewed:  
+---
+
+### Introduction
+
+In SerivcePulse 1.5.0 and higher screens to manage archived messages have been introduced in response to changes in ServiceControl relating to Archived Message retention and disposal. 
+
+ServiceControl version 1.12.0 and higher has a configurable [ErrorRetentionPeriod](/servicecontrol/creating-config-file.md) that schedules the disposal of archived failed messages.
+
+Archived messages can be found by going to the failed messages screen.
+
+![Archived Messages Tab](images/archive.png)
+
+### Archived Messages
+
+The Archived Messages page will open showing messages archived in the last 2 hours. To limit the set of displayed messages, select an option from the available predefined range.
+
+![Archive Filters](images/archive-filters.png)
+
+A scheduled task runs at a [configurable interval](/servicecontrol/how-purge-expired-data.md) to delete archived messages that have been in the archive for the [error retention period](/servicecontrol/creating-config-file.md) configured in ServiceControl. 
```

Comment:
I agree
On Sun, 3 Apr 2016 at 8:37 PM, Simon Cropp <notifications@github.com> wrote:

> In servicepulse/intro-archived-messages.md
> <https://github.com/Particular/docs.particular.net/pull/1376#discussion_r58308192>
> :
>
> > +
> > +In SerivcePulse 1.5.0 and higher screens to manage archived messages have been introduced in response to changes in ServiceControl relating to Archived Message retention and disposal.
> > +
> > +ServiceControl version 1.12.0 and higher has a configurable [ErrorRetentionPeriod](/servicecontrol/creating-config-file.md) that schedules the disposal of archived failed messages.
> > +
> > +Archived messages can be found by going to the failed messages screen.
> > +
> > +![Archived Messages Tab](images/archive.png)
> > +
> > +### Archived Messages
> > +
> > +The Archived Messages page will open showing messages archived in the last 2 hours. To limit the set of displayed messages, select an option from the available predefined range.
> > +
> > +![Archive Filters](images/archive-filters.png)
> > +
> > +A scheduled task runs at a [configurable interval](/servicecontrol/how-purge-expired-data.md) to delete archived messages that have been in the archive for the [error retention period](/servicecontrol/creating-config-file.md) configured in ServiceControl.
>
> ok. just saying "unarchive" sounds awkward
>
> —
> You are receiving this because you were mentioned.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/docs.particular.net/pull/1376/files/5706d434e963470fd5a5fcfb77db577fcb676152#r58308192>
>